ricochet.pro

```diff
@@ -61,6 +61,16 @@ contains(DEFINES, RICOCHET_NO_PORTABLE) {
         scalable_icon.files = icons/ricochet.svg
         INSTALLS += target shortcut icon scalable_icon
 
+        system(cp -f contrib/usr.bin.ricochet-apparmor contrib/usr.bin.ricochet)
+        QMAKE_CLEAN += contrib/usr.bin.ricochet
+        apparmor_profile.files = contrib/usr.bin.ricochet
+        !isEmpty(APPARMORDIR) {
+                apparmor_profile.path = $${APPARMORDIR}/
+        } else {
+                apparmor_profile.path = /etc/apparmor.d/
```

Comment:
Does this end up creating an /etc/apparmor.d/ on systems that don't use apparmor? I think ideal logic would be to install to `/etc/apparmor.d/` if that directory already exists, or install to `$APPARMORDIR` if explicitly specified, and otherwise not install.

One way to do this is to specify commands to execute in `apparmor_profile.extras`, instead of using `.files`. You could avoid the extra `cp` above that way too.